package.json

```
@@ -1,7 +1,8 @@
 {
   "name": "nuxt-ua",
-  "version": "1.0.8",
+  "version": "2.0.0",
```

```suggestion
  "version": "1.0.8",
```

To update version on `npm version` command automatically.